File: Exercício_15.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
 float peso, altura, r;
 printf("Digite o peso:");
 scanf("%f",&peso);

 printf("Digite a altura:");
 scanf("%f",&altura);

 r=peso/(altura*altura);

    if(r>20)
    {    
    printf("Abaixo do peso");
    }
    else
    {
     if(r>25)
       {
       printf("Acima do peso");
       }
     else
      {
      printf("Peso ideal");
      }
    }
 return 0;
}

File: Exercício_15.py
alt = float(input('Digite a sua altura em (m): '))
peso = float(input('Digite o seu peso (em Kg): '))
 
imc = peso/(alt*alt)
 
if imc < 20:
    print("Abaixo do peso!")
elif imc < 25:
    print("Peso ideal!")
else:
    print("Acima do peso!")